migration-guide.md

```diff
@@ -1,6 +1,16 @@
 akka-kryo-serialization - migration guide
 =========================================
 
+Migration from 1.1.x to 2.0.x
+-----------------------------
+
+* By moving to Kryo 5, it is unlikely to read bytes written with older versions. Please refer to Kryo's [Migrationto v5](https://github.com/EsotericSoftware/kryo/wiki/Migration-to-v5) guide for hints how to migrate data if necessary.
```

Comment:
By moving to Kryo 5 data created by a previous versions is unlikely to be readable with this version. ....

We should list the other breaking changes or if none that only kryo changed...

Reply:
I will remove the already deprecated `ScalaProductSerializer` (and some small cleanup) in another pull request